controllers/partner_merge_controller.py

```
@@ -0,0 +1,22 @@
+from odoo import http
+from odoo.http import request
+import json
```

Ruff (0.8.2)

`json` imported but unused

Remove unused import: `json`

(F401)